test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -15,26 +16,131 @@ var customMatchers = require('../assets/custom_matchers');
  *
  */
 
+var PLOT_DELAY = 200;
+var MOUSE_DELAY = 20;
+var MODEBAR_DELAY = 500;
 
-describe('Test plot structure', function() {
+
+describe('Test gl plot interactions', function() {
     'use strict';
 
+    var gd;
+
     beforeEach(function() {
         jasmine.addMatchers(customMatchers);
     });
 
-    afterEach(destroyGraphDiv);
+    afterEach(function() {
+        var fullLayout = gd._fullLayout,
+            sceneIds;
+
+        sceneIds = Plots.getSubplotIds(fullLayout, 'gl3d');
+        sceneIds.forEach(function(id) {
+            fullLayout[id]._scene.destroy();
```

Comment:
@mdtusz I think I found why running the gl interactions tests was so shaky and machine-dependent:

- destroying the graph div does not clear the gl context associated with it, meaning that many contexts were piling up on the test page making each tests slowing to run as time progressed.